src/color/opacify.js

```diff
@@ -32,7 +33,8 @@ import curry from '../internalHelpers/_curry'
  * }
  */
 function opacify(amount: number, color: string) {
-  const parsedColor = parseToRgb(color)
+  const normalizedColor = nameToHex(color)
```

Comment:
@bhough I think we can drop `nameToHex ` here since it's already done by default in `parseToRgb`. Same goes for all the other functions including the ones using `parseToHsl`